src/types/types.ts

```diff
@@ -12,3 +12,14 @@ export interface EntityType {
 }
 
 export type MapToEntity<Type extends EntityType> = MapPropsToDate<Type, "createdAt" | "updatedAt">;
+
+export enum LicenseType {
```

Comment:
I'd prefer avoiding `enum`.

Reply:
Moved from enum to the same as used in the API